relay-general/src/store/transactions.rs

```diff
@@ -57,23 +57,53 @@ pub fn validate_timestamps(
 /// "/user/123134/login".
 /// Newer SDK send the [`TransactionSource`] attribute, which we can rely on to determine cardinality,
 /// but for old SDKs, we fall back to this list.
-pub fn is_high_cardinality_sdk(client_sdk: &Annotated<ClientSdkInfo>) -> bool {
+pub fn is_high_cardinality_sdk(event: &Event) -> bool {
+    let client_sdk = match event.client_sdk.value() {
+        Some(info) => info,
+        None => {
+            return false;
+        }
+    };
+
     let sdk_name = client_sdk
+        .name
         .value()
-        .and_then(|sdk| sdk.name.value())
         .map(|s| s.as_str())
         .unwrap_or_default();
 
-    [
+    if [
         "sentry.javascript.angular",
         "sentry.javascript.browser",
         "sentry.javascript.ember",
         "sentry.javascript.gatsby",
         "sentry.javascript.react",
         "sentry.javascript.remix",
         "sentry.javascript.vue",
+        "sentry.javascript.nextjs",
+        "sentry.php.laravel",
+        "sentry.php.symphony",
     ]
     .contains(&sdk_name)
+    {
+        return true;
+    }
+
+    // TODO: Test with actual payloads
+    let is_http_status_404 = event.get_tag_value("http.status_code") == Some("404");
```

Comment:
Discover suggests this is a tag, however this is actually on spans, same for the method below.

Reply:
Taking this back, I was looking at JavaScript instead of Python payloads. In Python, this is being sent in by the SDK.